packages/components/src/components/EmulatedBoldText/EmulatedBoldText.module.scss

```diff
@@ -10,4 +10,7 @@
   height: 0;
   line-height: 0;
   visibility: hidden;
+
+  /* fixes focus outline */
+  margin-top: calc(var(--size-px--s) * -1);
```

Comment:
Bei welchem Element tritt das Problem auf?

Reply:
in der Header navigation

Reply:
```suggestion
  overflow-y: clip;
```

Als Idee?